src/components/BRB/BRBMainComponent.tsx

```diff
@@ -212,7 +212,7 @@ export const BRBMainComponent = () => {
           setIsAlertVisible={setIsAlertVisible}
         />
 
-        <Section padding="0 0 0 0">
+        <Section>
```

Comment:
All Sections need to have Content, BRB header should match width of relevant sections if content is covering them, this should be true for BRB Chat as well.